src/libguac/Makefile.am

```diff
@@ -160,7 +160,7 @@ libguac_la_CFLAGS = \
     -Werror -Wall -pedantic
 
 libguac_la_LDFLAGS =     \
-    -version-info 23:0:0 \
+    -version-info 24:0:0 \
```

Comment:
I'm late to the party here, but are the changes between 1.5.4 and 1.5.5 API-breaking?

If not, I think this should be `24:0:1` (interfaces added, not changed or removed). Doing a quick diff between `1.5.4` and `staging/1.5.5`, I'm not seeing anything stand out as breaking changes.

Reply:
Nah, this was probably my bad, here - I was trying to figure out the right one to do.

Also, do you mean `24:0:1`  or `23:0:1`?

I can open a new PR to fix it...

Reply:
Never mind - I think you were right to update to `24:0:0`. In the case of 1.5.5, the only change I see to any interface is the additional `socket_lock` member added to `guac_socket_ssl_data`, and that does qualify as a "changed interface".